test/navbar_router_test.dart

```diff
@@ -502,6 +697,10 @@ void main() {
 
   group('Test NavbarType: NavbarType.notched ', () {
     group('NavbarType.notched: should build destination and navbar items', () {
+      group('Badges test', () {
+        badgeGroupTest();
```

Comment:
Oh, so the tests are the same for each navbar type? I think they're a bit different for each of them so I just put the badgeTest in for each type.
If they're truly the same tests, I'll loop through the types and refractor my badge test!

Reply:
Yes kind of similar for each navbar type but,  it would not make any difference if you run the same tests (badeGroupTests) without passing a different navbartype for each of the test

Reply:
Hey, sorry for the delay I was dealing with schoolwork. I've added the code to include more navbar types for badge test and it's still working well! Consider this as resolve 😄

Reply:
This looks great!